app/src/main/java/com/example/bigowlapp/activity/ScheduleReportActivity.java

```diff
@@ -59,7 +56,12 @@ private void subscribeToData() {
                 scheduleReportTitle.setText(schedule.getTitle());
                 scheduleReportStartTime.setText(schedule.getStartTime().toDate().toString());
                 scheduleReportEndTime.setText(schedule.getEndTime().toDate().toString());
-                scheduleReportLocation.setText(GeoLocationFormatter.formatLocation(this, schedule.getLocation()));
+
```

Comment:
Flip/Invert the if statement in line 49 to reduce complexity. If you click on the if and then click lightbulb, and then click invert if  in Android Studio, it will do this automatically

Also if you have the time, you can add a test case that covers the `schedule = null` scenario

https://github.com/Landry333/Big-Owl/blob/b973729ce9a0ef0039a049b96c5be7683728a256/app/src/main/java/com/example/bigowlapp/activity/ScheduleReportActivity.java#L50-L54